src/couchbeam.erl

```diff
@@ -1292,6 +1357,22 @@ replicate_test() ->
     ?assert(is_binary(InstanceStartTime)),
     ok.
 
+config_test() ->
+    Server = couchbeam:server_connection(),
+    {ok, {AllConfig}} = couchbeam:get_config(Server),
+    ?assert(proplists:is_defined(<<"httpd">>, AllConfig)),
+    {ok, {HttpdConfig}} = couchbeam:get_config(Server, <<"httpd">>),
+    ?assert(proplists:is_defined(<<"port">>, HttpdConfig)),
+    {ok, Port} = couchbeam:get_config(Server, <<"httpd">>, <<"port">>),
+    ?assertEqual(<<"5984">>, Port),
+    ?assertMatch({ok, <<>>}, couchbeam:set_config(Server, <<"couchbeam">>,<<"test">>, <<"1">>, false)),
+    ?assertEqual({ok, <<"1">>}, couchbeam:get_config(Server, <<"couchbeam">>,<<"test">>)),
+    ?assertEqual({ok, <<"1">>}, couchbeam:set_config(Server, <<"couchbeam">>,<<"test">>, <<"2">>,  false)),
+    ?assertEqual({ok, <<"2">>}, couchbeam:get_config(Server, <<"couchbeam">>, <<"test">>)),
+    ?assertEqual({ok, <<"2">>}, couchbeam:delete_config(Server, <<"couchbeam">>, <<"test">>, false)),
+    ?assertEqual({error, not_found}, couchbeam:get_config(Server, <<"couchbeam">>,  <<"test">>)),
+    ok.
```

Comment:
Note: to properly count the tests:
1. suffix the function with `_test_` (`config_test_() ->`)
1. wrap these asserts in a list & prefix them with `_` (`[?_assertEqual(<<"5984">>, Port), ...]`)
1. remove that last `ok`